dependencies/graknlabs/repositories.bzl

```diff
@@ -42,8 +42,8 @@ def graknlabs_graql():
 def graknlabs_protocol():
     git_repository(
         name = "graknlabs_protocol",
-        remote = "https://github.com/graknlabs/protocol",
-        tag = "2.0.0-alpha-6", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_protocol
+        remote = "https://github.com/alexjpwalker/protocol",
+        commit = "e5f1a1046c2696d7d7301f5688d9e33e7a38283e", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_protocol
```

Comment:
(https://github.com/graknlabs/protocol/pull/112) Update to graknlabs before merging.